tests/Test.ADAL.NET.Unit/BrokerParametersTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal;
+using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Cache;
+using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.ClientCreds;
+using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Flows;
+using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Helpers;
+using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Instance;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Test.ADAL.NET.Common;
```

Comment:
Any reason using statements are not inside the namespace? (it's a best practice to do that)

Reply:
first reason is to be consistent with all other classes in the library, second - Visual Studio and ReSharper put it outside by default.

Reply:
I should have asked: What is the background for this decision? 

Aside: ReSharper defaults would likely want to put these inside (depending on version maybe).

Reply:
We use default settings of ReSharper and it put it outside. Also most of classes in .NET Foundation (for example - https://github.com/dotnet/coreclr/tree/master/src/mscorlib/src/System) use outside option